CONTRIBUTING.md

```diff
@@ -7,3 +7,31 @@ Pull requests will be evaluated by the repository guardians on a schedule and if
 
 All contributors retain the original copyright to their stuff, but by contributing to this project, you grant a world-wide, royalty-free, perpetual, irrevocable, non-exclusive, transferable license to all users **under the terms of the license under which this project is distributed.**
 
+## Git Workflow
+
+### Creating a Pull Request
+
+- Branch new features off the `develop` branch.
+- Naming convention based off the problem at hand.  (i.e. update-export-for-fault-tolerance )
+- Before pushing your new feature up to prod make sure to pull in the latest develop and fix any merge conflicts that might happen.
```

Comment:
The phrase "pushing your new feature up to prod" is a little confusing here since prod is a manual deployment currently. I don't think there is a way to push directly to prod.